src/number_theory/number_field/embeddings.lean

```diff
@@ -123,3 +125,134 @@ end
 end bounded
 
 end number_field.embeddings
+
+section place
+
+variables {K : Type*} [field K] {A : Type*} [normed_division_ring A] [nontrivial A] (φ : K →+* A)
+
+/-- An embedding into a normed division ring defines a place of `K` -/
+def number_field.place :
+  absolute_value K ℝ := absolute_value.comp (is_absolute_value.to_absolute_value (norm : A → ℝ))
+    φ.injective
+
+lemma number_field.place_apply (x : K) :
+  (number_field.place φ) x = norm(φ x) := by refl
+
+end place
+
+namespace number_field.complex_embedding
+
+open complex number_field
+
+open_locale complex_conjugate
+
+variables {K : Type*} [field K]
+
+/-- The conjugate of a complex embedding as a complex embedding. -/
+def conjugate (φ : K →+* ℂ) : K →+* ℂ := star φ
+
+@[simp]
+lemma conjugate_coe_eq (φ : K →+* ℂ) (x : K) : (conjugate φ) x = conj (φ x) := rfl
+
+lemma conjugate_conjugate_eq (φ : K →+* ℂ) :
+  conjugate (conjugate φ) = φ := star_involutive φ
```

Comment:
Now that you have the `star` spelling this feels sort of unnecessary.

Reply:
You still have these three definitions, but resolved this comment; did you mean to remove them?

bors r-
bors d+

Reply:
> You still have these three definitions, but resolved this comment; did you mean to remove them?

Well, I think I want to keep `def conjugate` because calling it `conjugate` is more natural than calling it `star`. (I guess we could do something like what is done for complex conjugation but I am not sure how to do that). I removed `conjugate_conjugate_eq` but `conjugate_coe_eq ` is useful is some proofs so I would like to keep it too